packages/server/src/cache/timeouts.ts

```diff
@@ -15,7 +16,8 @@ export enum types {
   UPDATE_PROFILE = 'UPDATE_PROFILE'
 }
 
-type PrefixType = Opaque<string>
+enum _prefixType {}
```

Comment:
```suggestion
const enum _prefixType {}
```
(prevents code from being emitted)